apps/server/src/modules/task-card/uc/task-card.uc.spec.ts

```diff
@@ -238,10 +271,16 @@ describe('TaskCardUc', () => {
 				})
 			);
 		});
+		it('should call task update to add id of task-card to task', async () => {
+			await uc.create(user.id, taskCardCreateParams);
+
+			expect(taskService.update).toBeCalled();
```

Comment:
would be nice to check toBeCalledWith but I see, getting the task id at this point is more tricky
perhaps joining this test this with the above from 243